jsonschema-validation.md

```diff
@@ -969,40 +883,6 @@ draft-bhutton-json-schema-01, June 2022,
 Hoehrmann, B., "Scripting Media Types", RFC 4329, DOI 10.17487/RFC4329, April
 2006, <<https://www.rfc-editor.org/info/rfc4329>>.
 
-## [Appendix] Keywords Moved from Validation to Core
```

Comment:
I'm not sure removing this section is related to extracting vocabularies, but I fully agree with removing it.

Reply:
Hm... I'm not sure.  It probably got caught up from one of my more aggressive change experiments I have locally.

Still, I agree it can be removed.  I'm not fussed about removing it if you're not.